src/components/bulkConcepts/bulkConceptTable.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const BulkConceptTable = (concept) => {
+  const {
+    concept: {
+      id,
```

Comment:
No type checking, kindly do the needful

Reply:
Thanks but eslint actually does not allow propType validation, so its ok here @emasys

Reply:
I see.